x-pack/plugins/apm/server/lib/traces/get_top_traces.ts

```diff
@@ -48,7 +49,8 @@ export async function getTopTraces(
           }
         },
         { term: { [PROCESSOR_EVENT]: 'transaction' } }
-      ]
+      ],
+      should: [{ term: { [TRANSACTION_SAMPLED]: true } }]
```

Comment:
This is just something we missed from before but unrelated to this ticket right? (Cool to do that here just making sure I understand)

Reply:
I sorta related to this ticket, since without this clicking on an item in Trace overview might take the user to a bucket without a sample on the transaction groups page - and thus the bucket won't be highlighted.

Reply:
But yeah, we should probably have added it in the beginning.